### ISSUE TITLE
Add axis labels in pyplot easily

### ISSUE THREAD

**Issue by ssunkara1** [bloomberg/bqplot#474]
It would be really nice to show how to add x_labels and y_labels in the Pyplot notebook, as well.
In `matplotlib` it's as easy as declaring
```python
plt.xlabel('string')
```
whereas the bqplot documentation on pyplot.plot of 

```
If a scale labeled 'x' is required
        for that mark, options['x'] contains optional keyword arguments for the
        constructor of the corresponding axis type.
```
is cryptic at best.
This would be much appreciated, thanks!

**Comment by maartenbreddels** [comment]
Indeed, was just sitting next to someone showing bqplot, and it's not a nice transition coming from matplotlib not having xlabel/ylabel.

**Comment by dmadeka** [comment]
@ssunkara1 @maartenbreddels Closed by #503
